shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/DatasetAttributes.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+package org.shanoir.ng.download;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.dcm4che3.data.Attributes;
+
+public class DatasetAttributes {
```

Comment:
Hi, as discussed, please verify if it is still in use this class.

Reply:
removed